UnrealPlugin/LinuxBuildsUE5.md

```diff
@@ -0,0 +1,62 @@
+# Setting up a Linux Dedicated Server on PlayFab
+
+The purpose of this guide is to describe all the steps needed to build and verify a Linux dedicated server. This was tested on both Unreal Engine 5 early access
+and Unreal Engine 4.26 but the following steps will be specific to Unreal Engine 5, early access.
+
+## Requirements
+* Docker on Windows. If it is not installed, you can follow the instructions here: [Docker Desktop on Windows](https://docs.docker.com/desktop/windows/install/)
+
+## Instructions
+
+There is an [Unreal guide](https://docs.unrealengine.com/4.27/en-US/SharingAndReleasing/Linux/GettingStarted/) about the various cross - compilation options, which you should review. For Unreal Engine 5, early access, the -v17 Toolchain has been tested to work.
+
+* Download the toolchain for your engine and run the executable. Please note that this will work only if your Unreal Engine build is from source.
+* After the installation is complete, go to environment variables and make sure this exists on your system variables: **LINUX_MULTIARCH_ROOT**.
+* In the directory of your source Unreal Engine build, run again Setup.bat and GenerateProjectFiles.bat.
+* Now you can go back to your Unreal Engine project, build it in Development Editor mode and open the Unreal Editor.
+* At this point, you should be able to package your project in Linux for both client and server in a directory of your choice. For Unreal Engine 5, in the Editor, click packaging on the top right of the screen and the Linux option should now be available to be selected.
+
+## Create Docker Image of the Linux Server.
+
+At this moment, in the directory chosen for packaging, you should be able to see a **Linux** directory if you packaged a Linux Client and a **LinuxServer** directory if you packaged a Linux Server, which is required for the following steps.
+In a terminal of your choice, follow these steps:
+* cd into the **LinuxServer** directory that was selected for packaging.
+* Create a file named: dockerfile
+* Edit the file with your editor of choice (vim, emacs etc.)
+* Copy - paste the content presented below. Make sure to replace [YourGameNameServer].sh with the name of your linux server executable.
+
+```bash
+FROM ubuntu:18.04
+
+# Unreal refuses to run as root user, so we must create a user to run as
+# Docker uses root by default
+RUN useradd --system ue
+USER ue
+
+EXPOSE 7777/udp
+
+WORKDIR /server
+
+COPY --chown=ue:ue . /server
+CMD ./YourGameNameServer.sh
```

Comment:
the content of the bash file is missing

Reply:
The bash file is made automatically by Unreal when doing a Linux Server build.
The name is based on your own project name.
Do you have any suggestions on how to say this more clearly and succinctly?

Reply:
ah sorry, in the original guide there was a sh file additionally, that set the permissions of the log folder so that the game can write to it.

Reply:
In our own testing in this pass, we did not identify those steps as necessary. We're just running the game bash file directly, and it seems to work as described.